synapse/rest/client/v2_alpha/room_keys.py

```diff
@@ -190,6 +190,11 @@ def __init__(self, hs):
         user_id = requester.user.to_string()
         version = parse_string(request, "version")
```

Comment:
I think you can provide the `required` parameter to `parse_string` to do the same thing, not 100% sure though.

Reply:
You are correct, good shout!